go.mod

```diff
@@ -77,10 +77,10 @@ require (
 	github.com/gogo/protobuf v1.3.2 // replaced
 	github.com/golang-jwt/jwt/v4 v4.5.0
 	github.com/google/btree v1.1.2
-	github.com/google/go-attestation v0.4.4-0.20220404204839-8820d49b18d9
+	github.com/google/go-attestation v0.4.4-0.20230613144338-a9b6eb1eb888
```

Comment:
There's quite a big, sneaky jump here. Seems alright but calling it out. @strideynet.